Source/Wit/Public/TTS/Experience/TtsExperience.h

```diff
@@ -94,6 +94,12 @@ class WIT_API ATtsExperience : public AActor, public ITtsServiceBase
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "TTS")
 	EWitRequestAudioFormat AudioType{EWitRequestAudioFormat::Wav};
 	
+	/**
+	* Should the response be streamed by Wit.ai
+	*/
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "TTS")
+	bool bStream{false};
```

Comment:
Minor naming. Does it make sense to call it bIsStreaming or bUseStreaming?

Reply:
`bUseStreaming` works for me, will update